index.js

```diff
@@ -0,0 +1,223 @@
+      </p>
+      <ul class="tools">
+        <li>${projectInfo[0].tools[0]}</li>
+        <li>${projectInfo[0].tools[1]}</li>
+        <li>${projectInfo[0].tools[2]}</li>
+      </ul>
+      <button class="popBtn show">See project</button>
+    </div>
+  </div>
+  </div>
+
+  <div class="card-body">
+  <div class="card-img swap">
+  <img src="${projectInfo[1].popupPhoto}" width="100%"/>       
+  </div>
+  <div>
+    <h2 class="card-header">${projectInfo[1].header}</h2>
+    <ul class="card-sub">
+      <li class="remove">${projectInfo[1].canopy}</li>
+      <div class="bullet-point"><img src="media/Counter.svg" alt="Bullet Point"></div>
+      <li>${projectInfo[1].backend}</li>
+      <div class="bullet-point"><img src="media/Counter.svg" alt="Bullet Point"></div>
+      <li>${projectInfo[1].year}</li>
+    </ul>
+    <div class="card-content">
+      <p>
+      ${projectInfo[1].content}
+      </p>
+      <ul class="tools">
+        <li>${projectInfo[1].tools[0]}</li>
+        <li>${projectInfo[1].tools[1]}</li>
+        <li>${projectInfo[1].tools[2]}</li>
+      </ul>
+      <button class="popBtn show">See project</button>
+    </div>
+  </div>
+  </div>
+
+  <div class="card-body">
+  <div class="card-img">
+  <img src="${projectInfo[2].popupPhoto}" width="100%"/>       
+  </div>
+  <div>
+    <h2 class="card-header">${projectInfo[2].header}</h2>
+    <ul class="card-sub">
+      <li class="remove">${projectInfo[2].canopy}</li>
+      <div class="bullet-point"><img src="media/Counter.svg" alt="Bullet Point"></div>
+      <li>${projectInfo[2].backend}</li>
+      <div class="bullet-point"><img src="media/Counter.svg" alt="Bullet Point"></div>
+      <li>${projectInfo[2].year}</li>
+    </ul>
+    <div class="card-content">
+      <p>
+      ${projectInfo[1].content}
+      </p>
+      <ul class="tools">
+        <li>${projectInfo[2].tools[0]}</li>
+        <li>${projectInfo[2].tools[1]}</li>
+        <li>${projectInfo[2].tools[2]}</li>
+        <li>${projectInfo[2].tools[3]}</li>
+      </ul>
+      <button class="popBtn show">See project</button>
+    </div>
+  </div>
+  </div>
+
+  <div class="card-body">
+  <div class="card-img swap">
+  <img src="${projectInfo[3].popupPhoto}" width="100%"/>       
+  </div>
+  <div>
+    <h2 class="card-header">${projectInfo[3].header}</h2>
+    <ul class="card-sub">
+      <li class="remove">${projectInfo[1].canopy}</li>
+      <div class="bullet-point"><img src="media/Counter.svg" alt="Bullet Point"></div>
+      <li>${projectInfo[3].backend}</li>
+      <div class="bullet-point"><img src="media/Counter.svg" alt="Bullet Point"></div>
+      <li>${projectInfo[3].year}</li>
+    </ul>
+    <div class="card-content">
+      <p>
+      ${projectInfo[3].content}
+      </p>
+      <ul class="tools">
+        <li>${projectInfo[3].tools[0]}</li>
+        <li>${projectInfo[3].tools[1]}</li>
+        <li>${projectInfo[3].tools[2]}</li>
+      </ul>
+      <button class="popBtn show">See project</button>
```

Comment:
- [ ] Kindly ensure that your pop up window follows the design guideline to attain pixel perfect design.

- Your design
<img width="298" alt="image" src="https://github.com/rbd3/Narson-Portfolio/assets/41163571/884cacf0-0be5-4330-9a8a-16911b999f93">

- Figma design
<img width="193" alt="image" src="https://github.com/rbd3/Narson-Portfolio/assets/41163571/e720f98c-897b-4d10-b658-b3da3d7a3349">